docs/build.gradle

```diff
@@ -213,7 +213,13 @@ task generateJavaDocs(type: Javadoc) {
     options.links("https://docs.oracle.com/en/java/javase/17/docs/api/")
     options.links("https://docs.opencv.org/4.x/javadoc/")
     options.addStringOption("tag", "pre:a:Pre-Condition")
-    options.addBooleanOption("Xdoclint/package:-edu.wpi.first.math.proto," +
+    options.addBooleanOption("Xdoclint/package:" +
```

Comment:
This will not check any linting rather then just missing, right? If so, seems like we'd want it more fine grained.

Reply:
Correct. `-Xdoclint/package` seems to disable all checks for the specified packages, and it doesn't seem customizable such that one can disable specific checks.

https://docs.oracle.com/en/java/javase/17/docs/specs/man/javadoc.html

JDK 21 lets you add `@SuppressWarnings("doclint:missing")` on specific classes.

https://docs.oracle.com/en/java/javase/21/docs/specs/man/javadoc.html